e2e/paths/enterApplicant_test.js

```diff
@@ -22,23 +22,24 @@ Scenario('Filling in the full section for enter applicants', (I, enterApplicantP
   I.continueAndSave();
   I.seeEventSubmissionConfirmation(config.applicationActions.enterApplicants);
   caseViewPage.selectTab(caseViewPage.tabs.casePeople);
-  I.seeAnswerInTab(1, 'Applicant', 'Name of applicant', 'John Smith');
-  I.seeAnswerInTab(2, 'Applicant', 'Name of person to contact', 'Jonathon Walker');
-  I.seeAnswerInTab(3, 'Applicant', 'Job title', 'Legal adviser');
-  I.seeAnswerInTab(1, 'Address', 'Building and Street', 'Flat 2');
-  I.seeAnswerInTab(2, 'Address', '', 'Caversham House 15-17');
-  I.seeAnswerInTab(3, 'Address', '', 'Church Road');
-  I.seeAnswerInTab(4, 'Address', 'Town or City', 'Reading');
-  I.seeAnswerInTab(5, 'Address', 'Postcode/Zipcode', 'RG4 7AA');
-  I.seeAnswerInTab(6, 'Address', 'Country', 'United Kingdom');
-  I.seeAnswerInTab(5, 'Applicant', 'Mobile number', '7000000000');
-  I.seeAnswerInTab(6, 'Applicant', 'Telephone number', '00000000000');
-  I.seeAnswerInTab(7, 'Applicant', 'Email', 'applicant@email.com');
-  I.seeAnswerInTab(8, 'Applicant', 'Payment by account (PBA) number', 'PBA1234567');
   I.seeAnswerInTab(1, 'Solicitor', 'Solicitor\'s full name', 'John Smith');
```

Comment:
does Solicitor appear on top?

Reply:
this is an unitended change if so. Applicant should be first.

Reply:
👍

Reply:
solicitor appears below